### DISCUSSION TITLE
List of open source projects that have been relicensed

### DISCUSSION THREAD

**Discussion by anajsana** [todogroup/ospology#480] [OSPO Discussions]
_This discussion thread is mirrored from the TODO Slack Channel and is anonymized_

I'm looking for a list of open source projects that have been relicensed. I think I remember someone here posting something like this and was hoping someone has a better memory and can point me in the right direction

**Comment by anajsana** [comment]
_anonymous parson replied_

I think all project from HashiCorp -> Terraform, Vault, Consult
Redis
ElasticSearch
MongoDB
CockroachDB
RedHat did some changes to licence of their Linux distro…but I can’t find what was it exactly

**Comment by anajsana** [comment]
_anonymous parson replied_

page 12 of https://opensource.org/delayed-open-source-publication has more

**Comment by anajsana** [comment]
_anonymous person replied_

some efforts @ https://en.wikipedia.org/wiki/List_of_formerly_open-source_or_free_software

**Comment by anajsana** [comment]
_anonymous person replied_

It would be wonderful for GitHub to document when a repo that was licensed as <L1> for at least a week gets relicensed under <L2> where either are OSI-approved.

I run into projects that change licensing that are not database platforms. For example TinyMCE is moving to GPL, which is going to cause some anxiety.

**Comment by anajsana** [comment]
_anonymous person replied_

@ XYZperson I'm curious whether this is something GH is thinking about? I could do some of this now by looking that the license file in the API, but something more easily discoverable might be interesting.

**Reply by anajsana** [reply]
reply: there are no plans for that AFAIK.  our API does not favour time-series data in general. but could you please drop an issue in todogroup/gh-issues with your use case ? i can ask around

**Reply by anajsana** [reply]
reply: Issue filed here: https://github.com/todogroup/gh-issues/issues/87

**Comment by jlprat** [comment]
Akka got relicensed

**Comment by anajsana** [comment]
there is a list at the bottom of this post: https://www.linuxfoundation.org/blog/how-open-source-foundations-protect-the-licensing-integrity-of-open-source-projects and I think @ XYZPerson put together in the past some sort of list on a GH repo? (I can't find it so I might be wrong)

**Comment by jamietanna** [comment]
In a more positive vein, https://github.blog/changelog/2024-05-13-dependabot-core-is-now-open-source-with-an-mit-license/ was going from open source (not Open Source) to Open Source

**Comment by geekygirldawn** [comment]
Thanks! I've created a dataset based on all of this data here for a project we're working on for CHAOSS: https://github.com/chaoss/wg-data-science/blob/main/dataset/license-changes/inputdata.csv

**Reply by gkunz** [reply]
Thanks, @geekygirldawn! Just added Vault.
